components/ProgressIndicator.js

```diff
@@ -100,20 +104,16 @@ class ProgressIndicator extends Component {
     });
 
   render() {
-    const { className, currentIndex, ...other } = this.props;
+    const { className, currentIndex, ...other } = this.props; // eslint-disable-line no-unused-vars
```

Comment:
Change: disable linting for unused `currentIndex` prop

Reply:
Another options is: https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings